electrumsv/app_state.py

```diff
@@ -116,6 +117,7 @@ def __init__(self, config: SimpleConfig, gui_kind: str) -> None:
         self.credentials = CredentialCache()
         self.headers: Headers | None = None
         self._headers_lock = threading.RLock()
+        self._last_headers_save: float|None = time.time()
```

Comment:
Were you planning on using this as part of a conditional statement somewhere? (i.e. if x amount of time has passed since `last_headers_save` then ...

Reply:
Debugging. If we need to we can have someone go in the console and get the value.